index.html

```diff
@@ -20,20 +20,217 @@
             </div>
         </div>
         </div>
-    </header>
-    <main>
+
         <section>
-            <h1>Nwannah  <br>Cynthia</h1>
+            <h2>Nwannah  <br>Cynthia</h2>
             <p>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. <br> 
                 If you like what you see and have a project you need coded, don’t hesitate and contact me</p>
             <div class="bot">
-            <input type="button" value="Start Collaboration ">
+                <button type="button" id="bom">Start Collaboration</button>
         </div>
-
     </section>
+    </header>
+
+    <main>    
     <div class="sectiontwo">
-        <!-- <img src="./asset/shape.svg" alt="A joined shape image"> -->
+        <img src="./asset/shape.svg" alt="">
+    </div>
+    <div class="pro">
+        <h1>projects</h1>
+    </div>
+    <div id="section-three">
+        <div class="lap">
+            <h1>welcome</h1>
+        <img src="./asset/laptop-image.png" alt="">
+    </div>
     </div>
+    <section class="section-plan">
+        <div class="plan">
+        <h4>My Project Portfolio</h4>
+        <ul id="ul">
+            <li>HTML/CSS</li>
+            <lI>Ruby on Rails</lI>
+            <li>Javascript</li>
+        </ul>
+            <div class="see">
+                <p>see this project <img src="./asset/arrowright.png" alt=""></p>
+            </div>
```

Comment:
- [ ]  Please add the pseudo-classes to these buttons requested on the Figma template. kindly implement in all buttons